### PR TITLE
Add i18n tags

### LINKED ISSUE #965: Add missing i18n tags to UI texts
Based on an UI text audit, there is quite a large number of unlocalised UI texts. Refer to this file for complete list: https://github.com/apinf/api-umbrella-dashboard/wiki/Unlocalised-UI-texts

# Definition of done
- [ ] Check that all of the texts  listed in https://github.com/apinf/api-umbrella-dashboard/wiki/Unlocalised-UI-texts have i18n tags.
- [ ] Add any missing i18n tags.
- [ ] Alphabetize i18n strings in en.i18n.js

Note: the task may also require solving autoform internationalisation.

### LINKED ISSUE #1278: Add missing i18n tokens/texts related to User Account form
Related to #965

Add missing i18n tokens/texts related to User Account form:

    - Change Password
      - Current Password
      - New Password
      - Confirm new Password
      - Change Password (button)

### LINKED ISSUE #1279: Add missing i18n tokens/texts related to Project Branding form
Related to #965

Add missing i18n tokens/texts related to Project Branding form:

    - Project logo
	- Choose file
	- Cover photo
	- Remove
	- Site title
	- Site abbreviation
	- Site slogan
	- Site footer
	- Choose your color theme
	- Social media
	  - Name
	  - Url
	- Update (button)

### LINKED ISSUE #1280: Add missing i18n tokens/texts related to Settings form
Related to #965

Add missing i18n tokens/texts related to Settings form: 

    - Api Umbrella
     - Host
     - API Key
     - Auth Token
     - Base URL
    - Api Documentation Editor
      - Enable apiDocumentation Editor
    - Elasticsearch
      - Host
    - Mail
      - Enabled
      - Mailgun Username
      - Mailgun Password
   - Github
    - Client ID
    - Secret
   - Additional
    - Contact Form E-mail Address
   - Save

### PR DESCRIPTION
**Pull request by Alapan**
Closes #1278 
Closes #1279 
Closes #1280

### CODE REVIEW

**Comment by Alapan** [comment]
@brylie I added some of the tags. For autoforms, I am not sure how to add tags for the dropdown options, such as for the Send Feedback form.

**Comment by brylie** [comment]
@Alapan great work! To help us better track the translation process, please also reference the sub-tasks of #965 that this completes. You can see the sub-tasks that are linked to #965 by scrolling down in the comments section and looking for the 'open' label next to related issues.

**Comment by brylie** [comment]
We can merge this PR when all related issues have been linked. E.g. add multiple 'Closes #issue' to the PR description, each 'Closes' on its own line:

```
Closes #123
Closes #234
Closes #345
...
```

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "dc": "^2.0.0-beta.30",
     "elasticsearch": "^11.0.1",
     "github": "^1.4.0",
-    "jquery": "<3.0.0",
+    "jquery": "^2.2.4",
```
Comment (jykae) [inline package.json]: Is there some breaking reason we are not using jquery 3.x ? Is something breaking?

**Comment by jykae** [comment]
@Alapan Reviewing.. Code looks good. I was just wondering one thing why we have limited jQuery to 2.x ? Is there something breaking when using 3.x ? But it is not blocking merging of this, if not sure.
I just run this after lunch, and see that everything functions nice.

**Comment by jykae** [comment]
@Alapan If interested to translate autoforms, we have this package in project, https://github.com/gwendall/meteor-autoform-i18n 
It might provide some help for what you were wondering.

**Comment by jykae** [comment]
Well done.

File: package.json (hunk 1 above)
Comment (brylie) [inline package.json]: We have to pin our jQuery version to less than 3, so that Bootstrap will work. Bootstrap does not support jQuery version 3+.

**Pull request merged**